company-ycmd.el

```diff
@@ -406,12 +406,12 @@ candidates list."
   (let* ((prefix-start-col (- (+ 1 (ycmd--column-in-bytes)) (length prefix)))
          (prefix-size (- start-col prefix-start-col))
          (prefix-diff (substring-no-properties prefix 0 prefix-size))
+         (prefix-diff-p (s-present? prefix-diff))
```

Comment:
This is the commit that introduced the `prefix-diff`: 5070d10c1323c56a7af706a93cafa8ca31efbd37

But there is no information about a particular case